core/src/test/kotlin/io/specmatic/core/DictionaryTest.kt

```diff
@@ -0,0 +1,46 @@
+package io.specmatic.core
+
+import io.specmatic.core.pattern.parsedJSONObject
+import io.specmatic.core.value.JSONObjectValue
+import org.assertj.core.api.Assertions.assertThat
+
+import org.junit.jupiter.api.Test
+
+// NOTE: Concrete values are actual values, not placeholders like (string) or (number).
+// For example, "ATYGHA" is a concrete value, not a placeholder.
+class DictionaryTest {
```

Comment:
We need a test somewhere of how generation will work when a dictionary is involved.